test_handler.go

```diff
@@ -114,4 +86,27 @@ func GetRunSHA(r *http.Request) (runSHA string, err error) {
         runSHA = runParam
     }
     return runSHA, err
+}
+
+// GetBrowserNames loads, parses and returns the set of names of browsers
+// which are to be included (flagged as initially_loaded in the JSON).
+// TODO(lukebjerring): Persist in memory.
+func GetBrowserNames() (browserNames []string, err error) {
+    var bytes []byte
+    if bytes, err = ioutil.ReadFile("browsers.json"); err != nil {
```

Comment:
Not in this PR but we should abstract the loading of `browsers.json` into a utility method since it's done in a few places in our Go code.

Reply:
Will do (think I already do that as part of a PR I'm working on)